### ISSUE TITLE
Gives the pretrained model in ImageNet？

### ISSUE THREAD

**Issue by momo1986** [opconty/keras-shufflenetV2#5]
Hello, thanks for you sharing.
I have the question, is there a model pre-trained on ImageNet Datasets provided or should I train from scratch?
Thanks & Regards!

**Comment by opconty** [comment]
Hi, Thanks for your attention, 
due to lack of computational resources. I didn't train this model on ImageNet Datasets.
hope this helpful.
Best.
